src/app/components/Navbar/index.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+
+import { Button } from '@radix-ui/themes';
+import { useState } from 'react'
+import { twJoin } from 'tailwind-merge'
+
+import Brand from "@/app/components/Brand"
+import NavLink from "@/app/components/NavLink"
+
+import Wallet from '../Wallet';
+
+const Navbar = () => {
+    const [state, setState] = useState(false)
+
+    const handleNavMenu = () => {
+        setState(!state)
+        document.body.classList.toggle("overflow-hidden")
+    }
+
+    return (
+        <header className='fixed top-0 w-full z-40'>
+            <nav className={twJoin("w-full md:static md:text-sm flex", state && "fixed z-10 h-full", "md:bg-[#F5F1EB] bg-transparent")}>
```

Comment:
as mentioned above, i dont think we should be bringing in landing page styling. the tailwinds overrides (eg `md:bg-[#F5F1EB`, `lg:w-[990px]`) forces the rest of the app to look a certain way when we should defer to the default sizing, scaling, color schemes, etc. If we want to create a theme, it should be done at the tailwinds config level. Otherwise we should leverage radix themes. Even better, we should use radix primitives with tailwinds classes so that there's some consistency.